bundles/org.openhab.binding.caddx/README.md

```diff
@@ -0,0 +1,246 @@
+# Caddx Binding
+
+The Caddx binding is used for communicating with the Caddx alarm panels
+
+It provides connectivity to the alarm panel via a RS-232 serial connection to the Caddx interface or directly to the NX8E.
+
+
+## Supported Things
+
+This binding supports the following Thing types
+
+| Thing      | Thing Type | Description                                                            |
+|------------|------------|------------------------------------------------------------------------|
+| bridge     | Bridge     | The  RS-232 interface.                                                 |
+| panel      | Thing      | The basic representation of the alarm System.                          |
+| partition  | Thing      | Represents a controllable area within the alarm system.                |
+| zone       | Thing      | Represents a physical device such as a door, window, or motion sensor. |
+| keypad     | Thing      | Represents a keypad.                                                   |
+
+## Discovery
+
+First the Caddx bridge must be manually defined. The serial port, baud rate and protocol have to be set correctly to match the respective configuration of the panel.
+After the bridge is manually added and available to openHAB, the binding will automatically start to discover partitions and zones and add them to the discovery inbox.
+
+Note:
+There is currently no support to discover the available keypads.
+
+## Binding Configuration
+
+No binding configuration required.
+
+## Thing Configuration
+
+The things can be configured either through the online configuration utility via discovery, or manually through the configuration file.
+The following table shows the available configuration parameters for each thing.
+
+| Thing     | Configuration Parameters                                                                       |
+|-----------|------------------------------------------------------------------------------------------------|
+| bridge    | `serialPort` - Serial port for the bridge - Required                                           |
+|           | `protocol` - Protocol used for the communication (Binary, Ascii) - Required - Default = Binary |
```

Comment:
Does the user know wether to choose Ascii or Binary? Should you give him some advise?

Reply:
Since you mentioned you can't see my comments anymore, I'm commenting on this, so you might see it again.